packages/console/src/log/LogView.tsx

```diff
@@ -226,36 +227,30 @@ class LogView extends PureComponent<LogViewProps, LogViewState> {
       wordWrap: 'on',
     });
 
-    // When find widget is open, escape key closes it.
-    // Instead, capture it and do nothing. Same for shift-escape.
-    this.editor.addCommand(monaco.KeyCode.Escape, () => undefined);
-    this.editor.addCommand(
-      // eslint-disable-next-line no-bitwise
-      monaco.KeyMod.Shift | monaco.KeyCode.Escape,
-      () => undefined
-    );
-
-    // Restore regular escape to clear selection, when editorText has focus.
-    this.editor.addCommand(
-      monaco.KeyCode.Escape,
-      () => {
-        const position = this.editor?.getPosition();
-        assertNotNull(position);
-        this.editor?.setPosition(position);
-      },
-      'findWidgetVisible && editorTextFocus'
-    );
-
-    this.editor.addCommand(
-      // eslint-disable-next-line no-bitwise
-      monaco.KeyMod.Shift | monaco.KeyCode.Escape,
-      () => {
-        const position = this.editor?.getPosition();
-        assertNotNull(position);
-        this.editor?.setPosition(position);
-      },
-      'findWidgetVisible && editorTextFocus'
-    );
+    // Override default Monaco keybindings for `escape` and `shift-escape`
+    [
+      [monaco.KeyCode.Escape],
+      [monaco.KeyMod.Shift | monaco.KeyCode.Escape], // eslint-disable-line no-bitwise
+    ].forEach(keybindings => {
+      // Monaco default behavior is for escape key to close the find widget.
+      // Instead, capture it and do nothing. Same for shift-escape.
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      MonacoUtils.disableKeyBindings(this.editor!, keybindings);
```

Comment:
You could use an `assertNotNull(this.editor)` above instead of doing a non-null assertion; the advantage of that is you would just need to do it once above these statements and don't need to disable the warning in the multiple spots here.
NOt a big deal either way